endpoint/aurora.go

```diff
@@ -31,6 +32,7 @@ type AuroraRPC struct {
 	maxPriorityFeePerGasCache      *primitives.Quantity
 	maxPriorityFeePerGasValidUntil time.Time
 	url                            string
+	mu                             sync.Mutex
```

Comment:
nitpick: this mutex is related to `maxPriorityFee` stuff specifically (not necessarily to whole "Aurora Client"), so might be better to name it with same prefix